README.md

```diff
@@ -13,7 +13,7 @@ Hiredis-cluster is a fork of Hiredis-vip, with the following improvements:
 * Support for SSL/TLS introduced in Redis 6
 * Support for IPv6
 * Support authentication using AUTH
-* Using CMake as build system
+* Using CMake as the primary build system, but optionally directly via Makefile
```

Comment:
Make is the build tool. (Makefile is the default file name used, like CMakeLists.txt for CMake.)